src/lib/media/manip.ts

```diff
@@ -147,24 +153,35 @@ interface DoResizeOpts {
 }
 
 async function doResize(localUri: string, opts: DoResizeOpts): Promise<Image> {
+  // We should run the first iteration outside of the loop. This is the only one that should resize the image and the
+  // loop can deal with lowering the quality if needed. Iterating on the resize causes problems on iOS
+  let resizeRes = await manipulateAsync(
+    localUri,
+    [{resize: {height: opts.height, width: opts.width}}],
+    {format: SaveFormat.JPEG, compress: 1},
+  )
+
```

Comment:
Seems like the reason we may have been using `react-native-image-resizer` was because of a weird issue on iOS with the image getting resized to the wrong dimensions. What seems to happen is that if we iterate more than once on resizing it, the dimensions get wonky.

Moving this outside of the loop and only iterating on the compression resolves that problem.